opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -82,19 +82,13 @@ message Span {
   // the same `trace_id`. The ID is a 16-byte array. An ID with all zeroes
   // is considered invalid.
   //
-  // This field is semantically required. Receiver should generate new
```

Comment:
I would leave "This field is semantically required.", and maybe even strenghten it to "This field is semantically required to always have a valid value."

Reply:
There is a sentence "This field is required" right below that, which I think is sufficient. I don't understand what's the difference between "required" and "semantically required".

Reply:
I overlooked this. I also don't understand the difference.